src/main/java/com/textmessenger/model/Post.java

```diff
@@ -0,0 +1,8 @@
+package com.textmessenger.model;
+
+public class Post {
```

Comment:
Please add @Data annotation to entities to automatically implement getters/setters etc